packages/packagers/html/src/HTMLPackager.js

```diff
@@ -47,13 +49,73 @@ export default new Packager({
     }, []);
 
     let {html} = await posthtml([
-      insertBundleReferences.bind(this, bundles)
+      insertBundleReferences.bind(this, bundles),
+      replaceInlineAssetContent.bind(this, bundleGraph, getBundleResult)
     ]).process(code);
 
     return {contents: html};
   }
 });
 
+async function getAssetContent(
+  bundleGraph: BundleGraph,
+  getBundleResult,
+  assetId
+): Promise<?Blob> {
+  let inlineBundle: ?Bundle;
+  bundleGraph.traverseBundles((bundle, context, {stop}) => {
```

Comment:
This could probably be optimized to traverse the graph once at the start of packaging, and make a map of inline asset keys to bundles. This way you'd avoid re-traversing the graph for each one. Probably not a big deal, but might be nice as a future improvement.

Reply:
I am good with adding this. However, it is possible we have inline bundles that are a part of a different html we are parsing, right? 

That could still be a better performance hit. 🤔